query-engine/connectors/sql-query-connector/src/query_builder/mod.rs

```diff
@@ -7,7 +7,7 @@ pub use write::*;
 use prisma_models::{RecordProjection, RecordProjectionExt};
 use quaint::ast::{Column, Comparable, ConditionTree, Query, Row, Values};
 
-const PARAMETER_LIMIT: usize = 10000;
+const PARAMETER_LIMIT: usize = 2000;
```

Comment:
Maybe we need that db specific at some point?

Reply:
Also having 2000 parameters in a query is a lot, having 10000 is even more...